docs/csharp/language-reference/operators/bitwise-and-shift-operators.md

```diff
@@ -60,51 +62,60 @@ The `<<` operator shifts its left-hand operand left by the number of bits define
 
 Because the shift operators are defined only for the `int`, `uint`, `long`, and `ulong` types, the result of an operation always contains at least 32 bits. If the left-hand operand is of another integral type (`sbyte`, `byte`, `short`, `ushort`, or `char`), its value is converted to the `int` type, as the following example shows:
 
-[!code-csharp-interactive[left shift with promotion](snippets/shared/BitwiseAndShiftOperators.cs#LeftShiftPromoted)]
+:::code language="csharp-interactive" source="snippets/shared/BitwiseAndShiftOperators.cs" id="LeftShiftPromoted":::
 
 ## Right-shift operator >>
 
 The `>>` operator shifts its left-hand operand right by the number of bits defined by its right-hand operand. For information about how the right-hand operand defines the shift count, see the [Shift count of the shift operators](#shift-count-of-the-shift-operators) section.
 
 The right-shift operation discards the low-order bits, as the following example shows:
 
-[!code-csharp-interactive[right shift](snippets/shared/BitwiseAndShiftOperators.cs#RightShift)]
+:::code language="csharp-interactive" source="snippets/shared/BitwiseAndShiftOperators.cs" id="RightShift":::
 
 The high-order empty bit positions are set based on the type of the left-hand operand as follows:
 
 - If the left-hand operand is of type `int` or `long`, the right-shift operator performs an *arithmetic* shift: the value of the most significant bit (the sign bit) of the left-hand operand is propagated to the high-order empty bit positions. That is, the high-order empty bit positions are set to zero if the left-hand operand is non-negative and set to one if it's negative.
 
-  [!code-csharp-interactive[arithmetic right shift](snippets/shared/BitwiseAndShiftOperators.cs#ArithmeticRightShift)]
+  :::code language="csharp-interactive" source="snippets/shared/BitwiseAndShiftOperators.cs" id="ArithmeticRightShift":::
 
 - If the left-hand operand is of type `uint` or `ulong`, the right-shift operator performs a *logical* shift: the high-order empty bit positions are always set to zero.
 
-  [!code-csharp-interactive[logical right shift](snippets/shared/BitwiseAndShiftOperators.cs#LogicalRightShift)]
+  :::code language="csharp-interactive" source="snippets/shared/BitwiseAndShiftOperators.cs" id="LogicalRightShift":::
+
+> [!NOTE]
+> Use the [unsigned right-shift operator](#unsigned-right-shift-operator-) to perform a *logical* shift on signed integer types. This is preferred to casting the signed type to an unsigned type and casting back after the shift.
+
+## Unsigned right-shift operator >>>
+
+The `>>>` operator, available starting in C# 11, performs a *logical* shift on all integer types. The high-order bits are always set to zero, regardless of the type of the left side argument. The following example demonstrates the difference between `>>` and `>>>` for a negative integer value:
```

Comment:
This looks to be the first introduction of `*logical*` shift. There seems to be no mention of `*arithmetic*` anywhere either or that unsigned types always use a `*logical*` shift.